### PR TITLE
fix/lib Clippy errors

### PR DESCRIPTION
**Pull request by dirvine**
This includes PR #83 from @frabrunelle

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @Fraser999

(maidsafe_highfive has picked a reviewer for you, use r? to override)

File: src/lib.rs [hunk 1]
```diff
@@ -1,24 +1,33 @@
 // Copyright 2015 MaidSafe.net limited.
 //
-// This SAFE Network Software is licensed to you under (1) the MaidSafe.net Commercial License,
-// version 1.0 or later, or (2) The General Public License (GPL), version 3, depending on which
+// This SAFE Network Software is licensed to you under (1) the MaidSafe.net
+// Commercial License,
+// version 1.0 or later, or (2) The General Public License (GPL), version 3,
+// depending on which
```
Comment (afck) [inline src/lib.rs]: In other crates, rustfmt doesn't put line breaks in all the comments. Or is this something new in Clippy?

**Comment by dirvine** [comment]
I just pushed a commit with the rustfmt toml. Should not change this but somehow it does not now alter the license lines. I tried to ignore with the rustfmt pragma, but it prevented format of the full file :-( 

Anyhow this looks better now I think.

File: src/lib.rs [hunk 2]
```diff
@@ -30,15 +31,19 @@
...
 //! // Construct an `LruCache` of `<u8, String>`s, limited by key count
 //! let max_count = 10;
-//! let lru_cache = LruCache::<u8, String>::with_capacity(max_count);
+//! let _lru_cache = LruCache::<u8, String>::with_capacity(max_count);
 //!
 //! // Construct an `LruCache` of `<String, i64>`s, limited by expiry time
 //! let time_to_live = ::std::time::Duration::from_millis(100);
-//! let lru_cache = LruCache::<String, i64>::with_expiry_duration(time_to_live);
+//! let _lru_cache = LruCache::<String,
+//! i64>::with_expiry_duration(time_to_live);
```
Comment (afck) [inline src/lib.rs]: It seems to have broken every line in the file that was longer than 80 characters, even though the limit is 100.

**Pull request closed without merging**